starter-code/HeterogeneousRecyclerView/app/src/main/java/com/charlesdrews/heterogeneousrecyclerview/FootballRecyclerViewAdapter.java

```diff
@@ -11,12 +11,54 @@
 
 import java.util.List;
 
+import javax.xml.transform.Templates;
+
 /**
  * Created by charlie on 11/14/16.
  */
 
 public class FootballRecyclerViewAdapter extends RecyclerView.Adapter {
+    private List<BaseFootballObject> mBaseFootballObjects;
+
+    public FootballRecyclerViewAdapter(List<BaseFootballObject> baseFootballObjects) {
+        mBaseFootballObjects = baseFootballObjects;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        if (viewType==1) {
```

Comment:
I recommend using a constant like `PLAYER` rather than just an int like `1` to represent the player view type. this will be more clear to other developers looking at your code.